handel/src/services/dynamodb/params-schema.json

```diff
@@ -0,0 +1,361 @@
+                "type"
+            ],
+            "additionalProperties": false,
+            "errorMessage": {
+                "required": {
+                    "name": "The 'name' field in the 'partition_key' section is required",
+                    "type": "The 'type' field in the 'partition_key' section is required"
+                },
+                "additionalProperties": "Invalid/Unknown property specified in the 'partition_key' section"
+            }
+        },
+        "sort_key": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "errorMessage": "The 'name' parameter must be a string"
+                },
+                "type": {
+                    "type": "string",
+                    "description": "The 'type' parameter may only be 'String' or 'Number'",
+                    "pattern": "^String|Number$",
+                    "errorMessage": "The 'type' parameter must be 'String' or 'Number'"
+                }
+            },
+            "additionalProperties": false,
+            "errorMessage": {
+                "additionalProperties": "Invalid/Unknown property specified in the 'sort_key' section"
+            }
+        },
+        "provisioned_throughput": {
+            "type": "object",
+            "properties": {
+                "read_capacity_units": {
+                    "oneOf": [
+                        {
+                            "type": "string",
+                            "pattern": "^(\\d+)(?:-(\\d+))?$"
+                        },
+                        {
+                            "type": "integer"
+                        }
+                    ],
+                    "errorMessage": "The 'read_capacity_units' must be either a number or a numeric range (ex: 1-100)"
+                },
+                "write_capacity_units": {
+                    "oneOf": [
+                        {
+                            "type": "string",
+                            "pattern": "^(\\d+)(?:-(\\d+))?$"
+                        },
+                        {
+                            "type": "integer"
+                        }
+                    ],
+                    "errorMessage": "The 'write_capacity_units' must be either a number or a numeric range (ex: 1-100)"
+                },
+                "read_target_utilization": {
+                    "type": "number",
+                    "minimum": 0,
+                    "maximum": 100,
+                    "errorMessage": "The 'read_target_utilization' parameter must be a percentage"
+                },
+                "write_target_utilization": {
+                    "type": "number",
+                    "minimum": 0,
+                    "maximum": 100,
+                    "errorMessage": "The 'write_target_utilization' parameter must be a percentage"
+                }
+            },
+            "additionalProperties": false,
```

Comment:
We'll want to have the "required" section here for "read_capacity_units" and "write_capacity_units"